magellan-library/src/main/java/com/wealthfront/magellan/navigation/NavigationTraverser.kt

```diff
@@ -71,22 +55,22 @@ public class NavigationTraverser(private val root: NavigableCompat) {
   }
 }
 
-public data class NavigationNode(val value: NavigableCompat, var children: List<NavigationNode>) {
-
-  val hasBackstack: Boolean
```

Comment:
nice to get rid of this "reflection" in favor of explicit overrides at the Navigable level